package.json

```diff
@@ -481,6 +480,7 @@
     "nock": "^13.2.9",
     "node-fetch": "^2.6.1",
     "nyc": "^15.0.0",
+    "playwright": "^1.29.2",
```

Comment:
Could you move `wait-on` as well? That is another `devDependency` that was added to `dependencies` in that PR